### PR TITLE
Test which shows that async program duplicates sync code at the moment

### PR DESCRIPTION
**Pull request by breathx**
Part of #293.

@gear-tech/dev

### CODE REVIEW

File: examples/sync_duplicate/src/lib.rs [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+static mut COUNTER: usize = 0;
+
+fn increase() {
+    unsafe {
+        COUNTER += 1;
+    }
+}
+
+fn get() -> usize {
+    unsafe { COUNTER.clone() }
```
Comment (shamilsan) [inline examples/sync_duplicate/src/lib.rs:15]: Why clone `Copy` type?

**Comment by NikVolf** [comment]
We need to employ special runner (executor) to mitigate this

File: examples/sync_duplicate/src/lib.rs (hunk 1 above)
Comment (breathx) [reply examples/sync_duplicate/src/lib.rs]: Removed

**Comment by NikVolf** [comment]
The problem is that we manually poll top level future in the `gstd_async::block_on`

**Comment by shamilsan** [comment]
Raised an issue: https://github.com/gear-tech/gear/issues/307

**Review by shamilsan — APPROVED** [review]

**Pull request merged**